src/desktop/apps/fair/routes.coffee

```diff
@@ -113,6 +113,9 @@ DEFAULT_CACHE_TIME = 60
     error: res.backboneError
     success: (show) -> res.redirect "/show/#{show.id}"
 
+@malformedFilterRedirect = (req, res, next) ->
+  res.redirect 301, "/#{req.params.id}/browse/artworks"
```

Comment:
Do we want to preserve the query params?

Reply:
This URLs are not user-facing so we just need to redirect until search engine indexes get updated with correct URLs so I think it doesn't matter.